pkgs/development/python-modules/ovirt-engine-sdk/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, pkgs }:
+
+buildPythonPackage rec {
+  pname = "ovirt-engine-sdk-python";
+  version = "4.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hz7rcz7l391n8ixcwvqw3iq5bx26fk9l5c6xxymv036zm3r9pnh";
+  };
+
+  doCheck = false;
+
+  buildInputs = with pkgs.python37Packages; [ 
+    pycurl 
+    six 
+    enum34
+    pkgs.libxml2
+  ];
+
+  prePatch = ''
+    substituteInPlace setup.py --replace "/usr/include/libxml2" "${pkgs.libxml2.dev}/include/libxml2"
```

in case dev output gets removed (unlikely)
```suggestion
    substituteInPlace setup.py --replace "/usr/include/libxml2" "${lib.getDev libxml2}/include/libxml2"
```